### PR TITLE
added presets for BIOMED2 IGK/IGL and TRB/TRD/TRG

### PR DESCRIPTION
**Pull request by mizraelson**

### CODE REVIEW

File: src/main/resources/presets/protocols/biomed2.yaml [hunk 1]
```diff
@@ -18,4 +18,36 @@
...
+    - type: SetSpecies
+      species: hs
+    - type: LeftAlignmentBoundaryNoPoint
+      floating: true
+    - type: RightAlignmentBoundaryNoPoint
+      floating: true
+      geneType: J
+    - type: MaterialTypeRNA
+
+"biomed2-human-trb-trd-trg":
```
Comment (dbolotin) [inline src/main/resources/presets/protocols/biomed2.yaml]:
```suggestion
"biomed2-human-tcr":
```
?

Comment (mizraelson) [reply src/main/resources/presets/protocols/biomed2.yaml]: no tra but maybe you are right?

Comment (dbolotin) [reply src/main/resources/presets/protocols/biomed2.yaml]: Is there a protocol for tra?

**Review by dbolotin — APPROVED** [review]

File: src/main/resources/presets/protocols/biomed2.yaml (hunk 1 above)
Comment (mizraelson) [reply src/main/resources/presets/protocols/biomed2.yaml]: no

**Pull request merged**